### PR TITLE
Add component for resource group

### PR DESCRIPTION
**Pull request by Flanker32**
Add component for resource group

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/ResourceGroupPanel.java [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+        cbResourceGroup.addPopupMenuListener(new PopupMenuListenerAdapter() {
+            @Override
+            public void popupMenuWillBecomeInvisible(PopupMenuEvent e) {
+                onSelectResourceGroup();
+            }
+        });
+    }
+
+    private void createResourceGroup() {
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/ResourceGroupPanel.java]: move private method last

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/ResourceGroupPanel.java [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        cbResourceGroup.setSelectedItem(null);
+        final NewResourceGroupDialog dialog = new NewResourceGroupDialog();
+        dialog.pack();
+        dialog.setLocationRelativeTo(this);
+        dialog.addWindowListener(new WindowAdapter() {
+            @Override
+            public void windowClosed(WindowEvent windowEvent) {
+                super.windowClosed(windowEvent);
+                final ResourceGroupWrapper newResourceGroup = dialog.getResourceGroup();
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/ResourceGroupPanel.java]: where is the validation part?

Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/ResourceGroupPanel.java]: Validation will be submitted in later PR

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/ResourceGroupPanel.java (hunk 1 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/ResourceGroupPanel.java]: Fixed

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/NewResourceGroupDialog.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+import javax.swing.KeyStroke;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+
+public class NewResourceGroupDialog extends JDialog {
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/NewResourceGroupDialog.java]: shall we have a baseDialog extends JDialog, with ok/cancel button

**Pull request merged**